Bottom: 7becaa0ae0cc3af16cd80f46321456792d004735
Top:    274477a847124e862d089b22912bb70d7eb669f3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:51:23 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/host_vars/bbworker/privileges.yml b/ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
similarity index 100%
rename from ansible-playbook/host_vars/bbworker/privileges.yml
rename to ansible-playbook/host_vars/buildbot-worker.novatech-llc.com/privileges.yml
